### PR TITLE
Beautify warning message

### PR DESCRIPTION
**Pull request by Pofigizm**
1. `rule` always undefined in this point, so I removed it
2.  Improved debugging by adding `meta`

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function jssNested() {
     return (match, name) => {
       const rule = container.getRule(name)
       if (rule) return rule.selector
-      warning(false, '[JSS] Could not find the referenced rule %s. \r\n%s', name, rule)
+      warning(false, '[JSS] Could not find the referenced rule %s in %s.', name, container.options.meta)
```
Comment (kof) [inline src/index.js]: meta is also an optional property, we need to think of something better

Comment (kof) [inline src/index.js]: we could use container, it has .toString method which will result in a a css string with all the rules, not sure though if that level of verbosity is nice

Comment (Pofigizm) [inline src/index.js]: I understand that `meta` is optional, but it will be better than we have now. (sure that not worse)

Comment (kof) [inline src/index.js]: actually we can use meta and container as a fallback: container.options.meta || container

**Comment by kof** [comment]
merged, thanks!

**Comment by kof** [comment]
will wait with a release if its not urgent for you

**Comment by Pofigizm** [comment]
thank you for jss ))

**Comment by kof** [comment]
Feel free to add your company here http://cssinjs.org/who-is-using , I started  this list just recently.

**Comment by Pofigizm** [comment]
Oh, sure, I will add our company to your list when we release the part of our project where we use jss. We are going to do it about two month later.

**Pull request merged**